entries/b/bankaust.com.au

```diff
@@ -0,0 +1,20 @@
+{
+  "Bank Australia": {
+    "domain": "bankaust.com.au.json",
+    "url": "https://www.bankaust.com.au",
+    "img": "bankaust.com.au.svg",
+    "tfa": [
+      "sms"
```

Comment:
```suggestion
      "custom-software"
```

Does Bank Australia support SMS MFA? If so, please attach some screenshots of the 2FA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.